src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,112 @@
+import CokoLogo from '@common/layout/CokoLogo';
+import MenuBar from '@common/layout/MenuBar';
+import { Layout, LeftSection, RightSection, Wrapper } from '@/style/style';
+import Header from '@common/layout/Header';
+import {
+  BadgeLabel,
+  JoinDateLabel,
+  LevelDiv,
+  MyProgressDiv,
+  MyQuizInfoDiv,
+  UserNameLabel,
+  ProfileSection,
+  BadgeSection,
+  LevelList,
+  LevelLabel,
+  MyCharacterImage,
+} from './styles';
```

Comment:
```suggestion
import * as S from "./styles";
```


. . .
```
//사용시
<S.BadeLabel />
<S.JoinDateLabel />
```
위와  같이 사용해 보는건 어떨까요?🧐

`S-dot`이라고 불리는 방식인데, 이런식으로 적으면 아래와 같은 이점이 있습니다.
- 해당 컴포넌트가 `styled-componet`라는것도 명확해짐
- 스타일을 작성하실 때, `S. `으로 접근이 가능하기 때문에 편리함
- import 부분을 간소화 할 수 있다.

생각해보시고, 적용해보시는 것을 추천드립니다!👏

Reply:
미니 프로젝트에서 적용했던 방법인데 프로젝트를 하다보니 이런 세세한 부분을 놓쳐버렸네요 😢
다시 복기시켜주셔서 감사합니다. 
@zzzRYT 님 의견대로  `import` 구문쪽에 간소화되는게 좋은 것 같습니다! 
꼼꼼하게 봐주셔서 감사합니다 😊

Reply:
26d1e68dc4d8d584ab8874a31dba00e9318d26f6 커밋에서 해당 방식을 적용시켰습니다. 확실히 `import` 구문이 간소화되네요 ㅎㅎ

Reply:
@zzzRYT 님께서 말씀하신 것처럼 해당 컴포넌트가 `styled-componet`라는 점, import 부분 간소화 등 장점이 확실한 방법입니다. 

아쉬운 점이라면 스타일드 컴포넌트 파일 안에 내가 필요로 하지 않는 스타일까지 있다면 `S-dot` 방식을 사용할 떄 그런 필요로 하지 않는 스타일까지 import 시키기 때문에 스타일드 컴포넌트 파일 자체가 조금 무거운 편이라면 약간의 시간차가 발생할 수 있다네용😊